wgsl/index.bs

```diff
@@ -8851,6 +8851,7 @@ shader that goes beyond the specified limits.
   </thead>
     <tr><td>Maximum number of members in a [=structure=] type<td>16383
     <tr><td>Maximum [=nesting depth=] of a [=composite=] type<td>255
+    <tr><td>Maximum nesting depth of statements in a function<td>127
```

Comment:
Nesting depth of statements isn't really defined.

It's mostly covered by compound_statement grammar element, but also: switch, loop, continuing each have their own handling.

Suggest changing this to

> Maximum nesting depth of brace-enclosed statements in a function.

That would mean, for example:

     fn foo() { /* depth 1*/  loop  { continuing { /*depth 3*/ break if true; }}} 

There's maybe an ambiguity about a switch body not being a statement? So then

> Maximum nesting depth of brace-enclosed statements or control flow in a function.

I don't know if that's better, or just confusing.   Your choice.